service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandlerTest.java

```diff
@@ -75,222 +99,292 @@
+                .build())
+            .build();
+    }
+
+    private Map<String, Object> getCaseDetailsWithMondayHearing() {
+        CaseData caseData = CaseData.builder()
+            .hearingDetails(wrapElements(HearingBooking.builder()
+                .startDate(MONDAY_DATE_AT_NOON)
+                .build()))
+            .build();
+
+        return mapper.convertValue(caseData, new TypeReference<>() {});
+    }
+
+    private StartEventResponse getStartEventResponse(CallbackRequest callbackRequest) {
+        return StartEventResponse.builder()
+            .caseDetails(callbackRequest.getCaseDetails())
+            .eventId(CASE_EVENT)
+            .token(TOKEN)
+            .build();
+    }
+
+    private CaseDataContent expectedCaseDataContent(CallbackRequest callbackRequest) {
+        return CaseDataContent.builder()
+            .eventToken(TOKEN)
+            .event(Event.builder()
+                .id(CASE_EVENT)
+                .build())
+            .data(callbackRequest.getCaseDetails().getData())
+            .build();
+    }
+
+    private void verifyCoreCaseDataApiIsCalledWithCorrectParameters() {
         verify(coreCaseDataApi).submitEventForCaseWorker(
-            TOKEN, AUTH_TOKEN, USER_ID, JURISDICTION, CASE_TYPE, CASE_ID, true, CaseDataContent.builder()
-                .eventToken(TOKEN)
-                .event(Event.builder()
-                    .id(CASE_EVENT)
-                    .build())
-                .data(callbackRequest.getCaseDetails().getData())
-                .build());
+            eq(TOKEN),
+            eq(AUTH_TOKEN),
+            eq(USER_ID),
+            eq(JURISDICTION),
+            eq(CASE_TYPE),
+            eq(CASE_ID),
+            eq(true),
+            caseDataContent.capture());
+    }
+
+    private CaseData getCaseData() {
+        return mapper.convertValue(caseDataContent.getValue().getData(), CaseData.class);
+    }
+
+    private List<Direction> localAuthorityDirections(CaseData caseData) {
+        return unwrapElements(caseData.getLocalAuthorityDirections());
+    }
+
+    private Direction expectedDirection(String directionText, LocalDateTime localDateTime) {
+        return Direction.builder()
+            .directionType(DIRECTION_TITLE)
+            .directionText(directionText)
+            .assignee(LOCAL_AUTHORITY)
+            .readOnly("No")
+            .directionRemovable("No")
+            .dateToBeCompletedBy(localDateTime)
+            .build();
+    }
+
+    private OrderDefinition getOrderDefinitionContainingTwoDirectionsWithDifferentDeltas() {
+        return OrderDefinition.builder()
+            .directions(ImmutableList.of(
+                directionConfiguration(DIRECTION_TEXT, TWO_DAYS_BEFORE_HEARING),
+                directionConfiguration(DIRECTION_TEXT, THREE_DAYS_BEFORE_HEARING)))
+            .build();
+    }
+
+    private DirectionConfiguration directionConfiguration(String directionText, String deltaValue) {
```

Comment:
Do we need both of these methods? can we not just have one with time, directionText, delta value passed in?

Reply:
seems silly to be passing in all that stuff when it's not needed, overloading methods isn't a bad thing :)

Reply:
ok makes sense :)